applications/fov-quicklook/templates/_helpers.tpl

```diff
@@ -0,0 +1,42 @@
+{{- define "fov-quicklook.env.s3_tile" -}}
+- name: QUICKLOOK_s3_tile
+  value: {{ .Values.s3_tile | toJson | quote }}
+- name: QUICKLOOK_s3_tile__access_key
+  valueFrom:
+    secretKeyRef:
+      name: fov-quicklook-secret
+      key: s3_tile_access_key
+- name: QUICKLOOK_s3_tile__secret_key
+  valueFrom:
+    secretKeyRef:
+      name: fov-quicklook-secret
+      key: s3_tile_secret_key
+{{- end }}
+
+{{- define "fov-quicklook.env.s3_repository" -}}
+- name: QUICKLOOK_s3_repository
+  value: {{ .Values.s3_repository | toJson | quote }}
+- name: QUICKLOOK_s3_repository__access_key
+  valueFrom:
+    secretKeyRef:
+      name: fov-quicklook-secret
+      key: s3_repository_acceess_key
```

Comment:
```suggestion
      key: s3_repository_access_key
```

Reply:
fixed in f0d243d

Reply:
fixed in 137df11551e17460ef6b3dcba721a23940aef9de